src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDeconstructTests.cs

```diff
@@ -5555,23 +5570,37 @@ static void Main()
 
             var discard1 = GetDiscardDesignations(tree).First();
             Assert.Null(model.GetDeclaredSymbol(discard1));
+            Assert.True(model.GetSymbolInfo(discard1).IsEmpty);
```

Comment:
Looks correct, a designation is not an expression, it doesn't have a type and doesn't refer to any symbol.

---
In reply to: [95036607](https://github.com/dotnet/roslyn/pull/16153#discussion_r95036607) [](ancestors = 95036607,95025285)